From ed08813fcdfb91b4863427f317dd9407ebb4a4b0 Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <abderrahim.kitouni@codethink.co.uk>
Date: Sat, 30 Jul 2022 15:17:14 +0200
Subject: [PATCH 1/3] Makefile.am: don't create /var/run/spice-vdagentd

/var/run is generally a symlink to /run, which is a tmpfs.
---
 Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 9c30bae..b368c82 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -153,9 +153,6 @@ gdmautostart_DATA = $(top_srcdir)/data/spice-vdagent.desktop
 gdmautostart2dir = $(datadir)/gdm/autostart/LoginWindow
 gdmautostart2_DATA = $(top_srcdir)/data/spice-vdagent.desktop
 
-install-data-local:
-	$(mkdir_p) $(DESTDIR)$(localstatedir)/run/spice-vdagentd
-
 if INIT_SCRIPT_RED_HAT
 initdir = $(sysconfdir)/rc.d/init.d
 init_SCRIPTS = $(top_srcdir)/data/spice-vdagentd
-- 
2.41.0

